joomla/joomla-cms
# [4.0] CPanel Option in one column

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/27490.

### Summary of Changes
Because CPanel  has unreadable fields in some windows sizes I changed the two column view to a one column view

### Testing Instructions

1. Open Global Configuration an see that all options displayed in two columns. 
2. Apply this patch
3. Open Global Configuration again and see, that the options displayed in one column. 

### Expected result

![Global Configuration   test   Administration](https://user-images.githubusercontent.com/9974686/73010503-d8737480-3e12-11ea-9ade-f72d3f3d6806.png)


### Actual result

![Global Configuration   test   Administration(1)](https://user-images.githubusercontent.com/9974686/73010518-ddd0bf00-3e12-11ea-935d-20537949d031.png)


### Documentation Changes Required

No
